openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RouteRequestHandler.java

```diff
@@ -504,4 +512,15 @@ private  boolean convertSetFlexibleMode(boolean useContractionHierarchies) throw
 
         return avoidCountryIds;
     }
+
+    private double validateMaximumSpeed(Double userSpeed) throws ParameterValueException {
+        double maximumSpeed = ((AppConfig.getGlobal().getServiceParameter("routing.profiles.default_params", "maximum_speed_lower_bound")) != null)
```

Comment:
See the comment in `MaximumSpeedCoreEdgeFilter`.

Reply:
How can I pass it to `RouteRequestHandler` from the `RouteProfileConfiguration` ?

Reply:
That's a fair point and I was wondering about it too. To me it it looks as a general design flaw of ORS. If this is the case it might be that the validation of the value provided in the query can be done only later in `ORSGraphHopper`.

Reply:
BTW: apart from verifying whether the `maximum_speed` parameter value is not below `maximum_speed_lower_bound` we need as check whether the parameter is compatible with the given profile. Unless the profile is car or hgv providing `maximum_speed` should raise an error. I don't think this is the case now.

Reply:
> BTW: apart from verifying whether the `maximum_speed` parameter value is not below `maximum_speed_lower_bound` we need as check whether the parameter is compatible with the given profile. Unless the profile is car or hgv providing `maximum_speed` should raise an error. I don't think this is the case now.

Yes I can fix it.So I should move the verification from `RouteRequestHandler` to `ORSGraphHopper` ?

Reply:
>So I should move the verification from RouteRequestHandler to ORSGraphHopper 

Actually, I was wrong about `ORSGraphHopper`, sorry for that. I think the verification should rather happen in `RoutingProfileManager` which has access to `RouteProfileConfiguration`.

Reply:
Do you know maybe in which function should I add it? Are all verifications done somewhere?

Reply:
You could add it for example [here](https://github.com/GIScience/openrouteservice/blob/b3eef42da7fe06c5965459a2ce501415adf46bff/openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfileManager.java#L578).